components/mdns/tests/host_test/make_eth2_netif

```diff
@@ -0,0 +1,5 @@
+sudo ip link add eth2 type dummy
```

Comment:
Is this to be bash script? Or is somewhere defined how to use this file?

Reply:
Add shebang line (and add sh extension)

Reply:
Removed the script, as these commands are listed in the `README` and the script is not used in the CI